src/math/Kconfig

```diff
@@ -9,6 +9,15 @@ config CORDIC_FIXED
 	  Select this to enable sin(), cos(), asin(), acos(),
 	  and cexp() functions as 16 bit and 32 bit versions.
 
+config LUT_TRIG_FIXED
+	bool "Lookup table based trigonometric functions"
+	default n
+	help
+	  Select this to enable sofm_lut_sin_fixed_32b() function. The
+	  calculation is using 1/4 wave lookup and interpolation.
+	  This option consumes 2052 bytes .bss RAM for the lookup
+	  table.
```

Comment:
Can we offer advice on when each trig type should be used. i.e. I would expect we export the same public API for all maths, but the internal calculations will depend on which Kconfig is selected by the user at build time.

Reply:
I added some text to Kconfig about preferring the lookup sine when used in hot code parts.